package.json

```diff
@@ -51,6 +51,7 @@
     "husky": "^3.0.5",
     "jest": "^24.9.0",
     "mkdirp": "^0.5.1",
+    "nock": "^12.0.3",
```

Comment:
This seems to be the most widely used tool to mock network requests in the node environment. It has very few dependencies as far as I've seen.

Reply:
Thanks for thinking through the cost of extra dependencies, and limiting the "node_modules" bloat.